Field.php

```diff
@@ -161,18 +168,17 @@ private function nextGeneration()
      * Start-Game function
      * Prints the field, calculates the next Generation and prints the new field for "maxSteps"-times.
      * Also, if a field is printed double and the field does'nt change anymore, the program will stop automatically.
+     * @param $_output BaseOutput
```

Comment:
Please also add a sentence as documentation to the parameter. (Example: The output that should be used to output the fields.)